modules/ROOT/pages/jms/jms-about.adoc

```diff
@@ -27,34 +23,28 @@ For information about each Mule 4 JMS Connector version, see:
 * xref:jms/jms-connector-reference-12x.adoc[JMS Connector v1.2.x Reference]
 * xref:jms/jms-connector-reference-11x.adoc[JMS Connector v1.1.x Reference]
 * xref:jms/jms-connector-reference-10x.adoc[JMS Connector v1.0.x Reference]
-====
+* xref:release-notes::connector/connector-jms.adoc[JMS Connector Release Notes] 
 
-Release Notes: xref:release-notes::connector/connector-jms.adoc[JMS Connector Release Notes] +
-Exchange: 
 
-== Queues - Point-to-point
+== Point-to-point Queues 
 
 In the point-to-point or queuing model, a sender posts messages to a particular queue and a receiver reads messages from the queue. Here, the sender knows the destination of the message and posts the message directly to the receiver’s queue.
 
-It is characterized by the following:
-
-* Only one consumer gets the message
-
-* The sender does not have to be running at the time the consumer consumes the message, nor does the consumer need to be running at the time the message is sent
-
-* Every message successfully processed is acknowledged by the consumer
-
-== Topics - Publish and subscribe
+The point-to-point queuing model is characterized by the following:
 
-The publish/subscribe model supports publishing messages to a particular message topic. Subscribers may register interest in receiving messages on a particular message topic. In this model, neither the publisher nor the subscriber know about each other. A good analogy for this is an anonymous bulletin board.
+* Only one consumer receives the message.
+* The sender does not need to be running at the time the consumer consumes the message, nor does the consumer need to be running at the time the message is sent.
+* Every message that is successfully processed is acknowledged by the consumer.
 
-JMS is characterized by the following:
+== Publish and Subscribe Topics
 
-* Multiple consumers (or none) receive the message
+The publish/subscribe model supports publishing messages to a particular message topic. Subscribers can register to receive messages on a particular message topic. In this model, neither the publisher nor the subscriber are aware of each other. An anonymous bulletin board is a good analogy for this concept.
 
-* There is a timing dependency between publishers and subscribers. The publisher has to create a message topic for clients to subscribe.
+The publish/subscribe model is characterized by the following:
 
-* The subscriber has to remain continuously active to receive messages, unless it has established a durable subscription. In that case, messages published while the subscriber is not connected redistribute when it reconnects.
+* Multiple consumers (or none) receive the message.
+* There is a timing dependency between publishers and subscribers. The publisher must create a message topic to which clients can subscribe.
```

Comment:
How are these two things--timing and ability to subscribe--related?
Should this be two list items?
If not, please reformat, anyway, to align with our style reference:

```suggestion
* There is a timing dependency between publishers and subscribers. 
+
+ The publisher must create a message topic to which clients can subscribe.
```

Reply:
Should the timing dependency be part of the FOLLOWING list item instead?